tensorboard/plugins/audio/summary.py

```diff
@@ -41,6 +42,11 @@
 audio = summary_v2.audio
 
 
+_LABELS_WARNING = (
+    "Labels on audio summaries are deprecated and will be removed."
```

Comment:
Maybe we should file an issue for this (to track the actual ultimate removal, including the actual UI elements) and include that here as a reference in case anyone notices and wants to object?

Reply:
Done: #3513.